packages/fxa-settings/src/models/contexts/AppContext.ts

```diff
@@ -24,13 +28,94 @@ export interface AppContextValue {
   config?: Config;
   account?: Account;
   session?: Session; // used exclusively for test mocking
+  uniqueUserId?: string; // used for experiments
+  experiments?: any; // TODO: add types for experiments
 }
 
 export interface SettingsContextValue {
   alertBarInfo?: AlertBarInfo;
   navigatorLanguages?: readonly string[];
 }
 
+/**
+ * Fetches or generates a new client ID that is stable for that browser client/cookie jar.
+ *
+ * N.B: Implemenation is taken from `fxa-content-server/.../models/unique-user-id.js` with
+ * inlined code that was written using Backbone utilities which could not immediately be transferred over.
+ * @returns a new or existing UUIDv4 for this user.
+ */
+function getUniqueUserId(): string {
+  function resumeTokenFromSearchParams(): string | null {
```

Comment:
We were only using resume tokens in content-server for email links. We don't have this any longer since switching to reset PW codes, so I don't think you'll ever get a value from this.

Reply:
I see a bunch of tests in auth server for the resume token. Since this is existing functionality, I'd feel more comfortable with keeping it so that there are fewer new things that could cause us to generate new IDs (to keep them stable).

I've filed FXA-11310 in the react cleanup epic to remove this eventually when the other resume token code is being removed too.